css/style.css

```diff
@@ -42,12 +39,120 @@ body {
   width: 100%;
   margin: auto;
   height: 500px;
+  background: url(../images/header_bg.jpg) no-repeat center top rgb(134, 141, 157);
+  background-size: cover;
+  }
+
+  .container2 {
+    width: 100%;
+    margin: auto;
+    height: 300px;
+    }
+
+header {
```

Comment:
You'll need to add additional white space so that the text doesn't hug the border of the screen. Try adding padding to each of the big blocks of content.